.changeset/soft-bananas-develop.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/starlight': patch
+---
+
+Add pinterest icon to the social links list"
```

Comment:
Suggesting a small tweak here to match out changeset [style guide](https://contribute.docs.astro.build/docs-for-code-changes/pull-requests/#changesets):
```suggestion
Adds Pinterest social icon
```

Reply:
Got it. I didn't know there was an Astro changeset doc.